modules/oe_content_publication/oe_content_publication.info.yml

```diff
@@ -5,12 +5,18 @@ package: OpenEuropa
 type: module
 core: 8.x
 dependencies:
+  - link:link
```

Comment:
Please add dependency  `oe_media_avportal:oe_media_avportal` - is needed for Thumbnail field.
Also would be good to sort it alphabetically for easy reading.

Reply:
We need to enable all new dependencies in post_update before creation of new fields. 

It can be done in 00003 https://github.com/openeuropa/oe_content/blob/3b5abe2ebfe5a72d5186b3ecd0611d73dcc28400/modules/oe_content_publication/oe_content_publication.post_update.php#L44.

Reply:
I added the ` oe_media_avportal` dependency, sorted the dependencies alphabetically and updated the `post_update` hook.